web_poet/page_inputs/annotated.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import Any, Tuple
+
+
+@dataclass
+class AnnotatedResult:
```

Comment:
AnnotatedInstance? I think ...Result made sense in scrapy-poet, as it was a result of a provider, but here it doesn't sound right.